### PR TITLE
Describe the TagBox demo overview

### PR DESCRIPTION
**Pull request by czernous-devexpress**

### CODE REVIEW

File: JSDemos/Demos/TagBox/Overview/description.md [hunk 1]
```diff
@@ -1 +1,36 @@
-The TagBox component is&nbsp;an&nbsp;editor that allows an&nbsp;end user to&nbsp;select multiple items from a&nbsp;drop-down list.
\ No newline at end of file
+The [TagBox](/Documentation/ApiReference/UI_Components/dxTagBox/) allows users to select multiple items from a drop-down list. This demo illustrates the following TagBox properties:
+
+- [items](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/items/)  
+  An array of items displayed by the TagBox.
+
+- [searchEnabled](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#searchEnabled)  
+  Specifies whether the search is enabled
```
Comment (AlexJameson) [inline JSDemos/Demos/TagBox/Overview/description.md]:
```suggestion
  Specifies whether the search is enabled.
```

File: JSDemos/Demos/TagBox/Overview/description.md [hunk 2]
```diff
@@ -1 +1,36 @@
...
+
+- [showSelectionControls](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#showSelectionControls)  
+  Defines whether selection controls are displayed
```
Comment (AlexJameson) [inline JSDemos/Demos/TagBox/Overview/description.md]: I suggest you to remove the passive voice. I also think that we can be more precise here and say 'selection checkboxes' to clarify what kind of controls we use.

```suggestion
  Defines whether to display the selection controls.
```

File: JSDemos/Demos/TagBox/Overview/description.md [hunk 3]
```diff
@@ -1 +1,36 @@
...
+
+- [applyValueMode](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#applyValueMode)  
+  Specifies whether selected values are applied instantly or when a user clicks the **OK** button.
+
+- [hideSelectedItems](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#hideSelectedItems)  
+  Specifies whether items are removed from the drop-down list upon selection.
```
Comment (AlexJameson) [inline JSDemos/Demos/TagBox/Overview/description.md]:
```suggestion
  Specifies whether the selected items are removed from the drop-down list.
```

File: JSDemos/Demos/TagBox/Overview/description.md [hunk 4]
```diff
@@ -1 +1,36 @@
...
+
+- [multiline](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#multiline)  
+  Displays selected items on single or multiple lines.
+
+- [acceptCustomValue](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#acceptCustomValue)  
+  Specifies whether users can enter custom values. Requires the [onCustomItemCreating](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#onCustomItemCreating) implementation.
```
Comment (AlexJameson) [inline JSDemos/Demos/TagBox/Overview/description.md]: We should not use articles before the property names alone. I see that you also use articles with the component names, which is strange to me, but I also see that Roman and our correctors are OK with it. Anyway, let's change the sentence as follows:

```suggestion
  Specifies whether users can enter custom values. Requires that you also implement the [onCustomItemCreating](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#onCustomItemCreating) handler.
```

File: JSDemos/Demos/TagBox/Overview/description.md [hunk 5]
```diff
@@ -1 +1,36 @@
...
+
+- [placeholder](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#placeholder)  
+  Customizes the TagBox placeholder.
+
+- [value](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#value)  
+  Specifies items the TagBox displays.
```
Comment (AlexJameson) [inline JSDemos/Demos/TagBox/Overview/description.md]: This description says the same thing as the `items` description. It seems like you should change one of them or both.

File: JSDemos/Demos/TagBox/Overview/description.md [hunk 6]
```diff
@@ -1 +1,36 @@
...
+
+- [disabled](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#disabled)  
+  Defines whether the component resopnds to user interaction.
```
Comment (AlexJameson) [inline JSDemos/Demos/TagBox/Overview/description.md]:
```suggestion
  Defines whether the component responds to user interaction.
```

File: JSDemos/Demos/TagBox/Overview/description.md [hunk 7]
```diff
@@ -1 +1,36 @@
...
+
+- [dataSource](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#dataSource)  
+  Binds the TagBox to data. Unlike the **items** property, **dataSource** accepts the [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) object that allows users to sort, filter, group, and shape data.
```
Comment (AlexJameson) [inline JSDemos/Demos/TagBox/Overview/description.md]: I would say that wee need to show more differences from the `items` property in terms of data binding. Maybe it is also a good idea to say that we cannot use these properties simultaneously.

File: JSDemos/Demos/TagBox/Overview/description.md [hunk 8]
```diff
@@ -1 +1,36 @@
...
+- [itemTemplate](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#itemTemplate)
```
Comment (AlexJameson) [inline JSDemos/Demos/TagBox/Overview/description.md:36]:
```suggestion

- [itemTemplate](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#itemTemplate)  
```

File: JSDemos/Demos/TagBox/Overview/description.md [hunk 9]
```diff
@@ -1 +1,36 @@
...
+- [value](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#value)  
+  Specifies items the TagBox displays.
+
+- [disabled](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#disabled)  
+  Defines whether the component resopnds to user interaction.
+
+- [dataSource](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#dataSource)  
+  Binds the TagBox to data. Unlike the **items** property, **dataSource** accepts the [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) object that allows users to sort, filter, group, and shape data.
+- [itemTemplate](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#itemTemplate)  
+  Allows you to customize the text field and drop-down list items.
```
Comment (AlexJameson) [inline JSDemos/Demos/TagBox/Overview/description.md]: The [fieldTemplate](https://js.devexpress.com/Documentation/21_1/ApiReference/UI_Components/dxTagBox/Configuration/#fieldTemplate) property allows users to customize the text field.

**Review by AlexJameson — CHANGES_REQUESTED** [review]

File: JSDemos/Demos/TagBox/Overview/description.md (hunk 9 above)
Comment (czernous-devexpress) [reply JSDemos/Demos/TagBox/Overview/description.md]: itemTemplate is used in this demo https://js.devexpress.com/Demos/WidgetsGallery/Demo/TagBox/Overview/jQuery/Light/

File: JSDemos/Demos/TagBox/Overview/description.md (hunk 5 above)
Comment (czernous-devexpress) [reply JSDemos/Demos/TagBox/Overview/description.md]: Why reinvent the wheel? The description here is straight to the point while the docs expand on it further

Comment (AlexJameson) [reply JSDemos/Demos/TagBox/Overview/description.md]: I totally agree. But this description is almost the same as 'An array of items displayed by the TagBox.' from the first point. It is hard to understand the difference. I think the following change will make this property's purpose more clear:

```suggestion
  Specifies items the TagBox currently displays.
```

File: JSDemos/Demos/TagBox/Overview/description.md (hunk 9 above)
Comment (AlexJameson) [reply JSDemos/Demos/TagBox/Overview/description.md]: Right you are. But does it customize the text field?

**Review by AlexJameson — DISMISSED** [review]

File: JSDemos/Demos/TagBox/Overview/description.md [hunk 10]
```diff
@@ -1 +1,36 @@
...
+
+- [value](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#value)  
+  Specifies items the TagBox currently displays.
+
+- [disabled](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#disabled)  
+  Defines whether the component responds to user interaction.
+
+- [dataSource](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#dataSource)  
+  Binds the TagBox to data. Unlike the **items** property, **dataSource** accepts the [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) object that allows users to sort, filter, group, and shape data. Note that you cannot use **items** and **dataSource** simultaneously.
+- [itemTemplate](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#itemTemplate)
```
Comment (RomanTsukanov) [inline JSDemos/Demos/TagBox/Overview/description.md:36]:
```suggestion

- [itemTemplate](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#itemTemplate)  
```

**Review by RomanTsukanov — APPROVED** [review]

**Pull request merged**